client/state/preferences/actions.js

```diff
@@ -93,11 +90,8 @@ export const savePreference = ( key, value ) => ( dispatch ) => {
 		},
 	};
 
-	return wpcom
-		.undocumented()
-		.me()
-		.preferences()
-		.update( payload )
+	return wpcom.req
+		.put( '/me/preferences', payload )
```

Comment:
Nit: I think we could use `.post()` directly in this case, looking at the `.put()` method and the browser devtools Network tab it looks like a regular `POST` request. Not sure what's the intention here so leaving it as a nit.

Reply:
The server endpoints use `POST` as a method. I'm not sure if `PUT` would even work.

Reply:
Yep. I'm just refactoring it to what we were using under the hood: https://github.com/Automattic/wp-calypso/pull/53697/files#diff-fe438575a1dba9db3a80ea27ae6e7267ae6e4345e14d35ed1fe072bd08fdb8b6L24

One step at a time.

In a subsequent PR we might want to completely get rid of `.put()` and replace it with `.post()` which essentially it already is.